downloader/history_downloader/history_manager.py

```diff
@@ -0,0 +1,19 @@
+import ccxt
+
+
+class HistoryDownloader:
+
+    def __init__(self, exchange_name: str, crypto_symbol: str):
+        """ init
+            Paramaters
+            ------------
+            exchange_name: str required - name of cryptocurrency, e.g. 'binance'
+            crypto_symbol: str required - symbol of cryptocurrency, e.g. in Bitcoin in Binance is 'BTC/USDT'
+        """
+        self.exchange_name = exchange_name.lower()
+        self.crypto_symbol = crypto_symbol
+        self.exchange = getattr(ccxt, self.exchange_name)()  # creating instance of binance exchange

```

Comment:
Interesting implementation ;) if it is working fine then it is good. Props for comment.

Reply:
Now I see that the comment should be "creating an instance of exchange", not necessarily binance, but that's only detail haha.